code/game/gamemodes/miniantags/revenant/revenant_abilities.dm

```diff
@@ -192,31 +196,28 @@
 	return TRUE
 
 //Overload Light: Breaks a light that's online and sends out lightning bolts to all nearby people.
-/obj/effect/proc_holder/spell/aoe_turf/revenant/overload
+/obj/effect/proc_holder/spell/aoe/revenant/overload
 	name = "Overload Lights"
 	desc = "Directs a large amount of essence into nearby electrical lights, causing lights to shock those nearby."
-	base_cooldown = 200
-	stun = 30
+	base_cooldown = 20 SECONDS
+	stun = 3 SECONDS
 	cast_amount = 45
 	var/shock_range = 2
-	var/shock_damage = 20
+	var/shock_damage = 40
 	action_icon_state = "overload_lights"
+	aoe_range = 5
 
-/obj/effect/proc_holder/spell/aoe_turf/revenant/create_new_targeting()
-	var/datum/spell_targeting/aoe/turf/T = new()
-	T.range = 5
-	return T
+/obj/effect/proc_holder/spell/aoe/revenant/overload/create_new_targeting()
+	var/datum/spell_targeting/aoe/obj/targeting = new()
```

Comment:
I think it makes more sense here to just have a `spell_targeting/aoe` and then override the `allowed_type` to be `/obj/machinery/light`. Why select all objects when you can also just instantly select only lights